packages/medusa/src/api/routes/admin/products/list-products.ts

```diff
@@ -241,16 +241,21 @@ export default async (req, res) => {
 
       let products: (Product | PricedProduct)[] = rawProducts
 
-      const includesPricing = ["variants", "variants.prices"].every(
+      // We only set prices if variants.prices are requested
+      const shouldSetPricing = ["variants", "variants.prices"].every(
```

Comment:
Suggestion(non blocking): should we have an util for checking that? Since it is used accross multiple places. Something like containsAll(collection, shouldContains)?

Reply:
In this case, I prefer the verbosity over a util, so will stick with this 👍